lib/events.js

```diff
@@ -148,7 +148,7 @@ EventEmitter.prototype.emit = function emit(type) {
 
   // If there is no 'error' event listener then throw.
   if (doError) {
-    er = arguments[1];
+    er = arguments.length <= 1 ? undefined : arguments[1];
```

Comment:
I think I'd rather just see an `if` like:

```js
if (arguments.length > 1)
  er = arguments[1];
```

since `er` is `undefined` by default.